api/v1beta1/tempest_types.go

```diff
@@ -177,7 +177,7 @@ type TempestRunSpec struct {
 	// +kubebuilder:validation:Optional
 	// Extra images that should be downloaded inside the test pod and uploaded to
 	// openstack.
-	ExtraImages []ExtraImagesType `json:"extraImages"`
+	ExtraImages []ExtraImagesType `json:"extraImages,omitempty"`
```

Comment:
This is needed otherwise the values will be set to null on the side of the webhook. It's nice this is fixed!